src/content/en/fundamentals/security/csp/index.md

```diff
@@ -3,7 +3,7 @@ book_path: /web/fundamentals/_book.yaml
 description: Content Security Policy can significantly reduce the risk and impact of cross-site scripting attacks in modern browsers.
 
 {# wf_published_on: 2012-06-15 #}
-{# wf_updated_on: 2016-02-19 #}
+{# wf_updated_on: 2017-08-11 #}
```

Comment:
(I can't comment on the image, but we should totally update it at some point. :) )